lang/en/tool_heartbeat.php

```diff
@@ -52,6 +52,11 @@
 $string['checkrangerequestok'] = 'Range requests are working, 206 response with only 10 bytes of data';
 $string['checkrangerequestbad'] = 'Range requests are bad! HTTP {$a->code} response with only {$a->bytes} bytes of data for {$a->url}';
 $string['ips_combine'] = 'The IPs listed above will be combined with the IPs listed below.';
+$string['errorascritical'] = ' Report check errors as:';
+$string['errorascritical_desc'] = 'This setting controls what check API errors are reported as in Nagios. "CRITICAL" is the most noisy, and "WARNING" is the least noisy for monitoring endpoints. Business hours is 9AM - 5PM in the server timezone.';
+$string['error_critical'] = 'CRITICAL';
```

Comment:
Are these strings in core?

Reply:
Not in the all caps case, and I wanted to make it clear that this directly to the nagios code